src/searchee.ts

```diff
@@ -19,6 +20,39 @@ export interface Searchee {
 	length: number;
 }
 
+function getFilesFromPath(dirPath): File[] {
```

Comment:
naming suggestion: `getFilesFromDataRoot(root: string)`

`dirPath` is slightly misleading because we're not sure if it's a directory or not yet